Java/TelemetrySystem/src/test/java/tddmicroexercises/telemetrysystem/TelemetryDiagnosticControlsTest.java

```diff
@@ -1,13 +1,35 @@
 package tddmicroexercises.telemetrysystem;
 
+import org.junit.Assert;
 import org.junit.jupiter.api.Test;
 
 public class TelemetryDiagnosticControlsTest
 {
-    
+    ConnectionClass connectionClass = new ConnectionClass();
+    ClientClass clientClass = new ClientClass();
+    TelemetryDiagnosticControls telemetryDiagnosticControls = new TelemetryDiagnosticControls(clientClass, connectionClass);
 	@Test
-    public void CheckTransmission_should_send_a_diagnostic_message_and_receive_a_status_message_response()
-    {
-    }
+    void checkTransmission() {
+        try {
+            if(clientClass.getOnlineStatus() == false)
+            {
+                throw new Exception("Unable to connect.");
+            }
+            telemetryDiagnosticControls.checkTransmission();
+            Assert.assertEquals(false, clientClass.getOnlineStatus());
 
+        } catch (Exception e) {
+            Assert.assertNotEquals("Unknown exception", e);
+        }
+    }
+    @Test
+    void checkGetDiagnostic(){
+       String dignosticInfo =  telemetryDiagnosticControls.getDiagnosticInfo();
+       Assert.assertEquals(dignosticInfo,telemetryDiagnosticControls.getDiagnosticInfo());
+    }

```

Comment:
this Test case is doesn't make sense
Remove it

Reply:
Removed the Test case